src/hazelcore/Action_Exp.re

```diff
@@ -632,6 +632,11 @@ and syn_perform_block =
   /* Backspace & Delete */
   // Handle 2 special cases for CommentLines
   // Case 1
+  // e.g.,
+  //  #  comment 1
+  //  #| comment 2
```

Comment:
only seeing this behavior for blocks in synthetic position -- still has the previous behavior in analytic position